### ISSUE TITLE
Creation of vxlan-vport in fastdp does not fail if udp:6784 is taken

### ISSUE THREAD

**Issue by brb** [weaveworks/weave#2375]
To reproduce:

```
$ weave reset
$ nc -l -u -p 6784 &
$ weave launch
```

In the case of failure, we would expect to see a warning https://github.com/weaveworks/weave/blob/v1.5.2/router/fastdp.go#L405 , but it doesn't happen.

It might be a case that the creation of vxlan-vport is async in the kernel, because no error is returned. However, in the dmesg report we can spot it:
```
[175079.718300] vxlan: Cannot bind port 6784, err=-98
```

Kernel version: 4.5.4-1-ARCH

**Comment by brb** [comment]
The issue does not apply to the 4.2.0 kernel.

**Comment by rade** [comment]
@brb Can you think of any way of fixing this? Checking the port beforehand would be racy.

**Comment by brb** [comment]
http://marc.info/?l=linux-netdev&m=147075637009483&w=2

In order to detect such case, we could check whether a corresponding netdev is in the `DOWN` state after a vxlan vport has been successfully created.

**Comment by rade** [comment]
> In order to detect such case, we could check whether a corresponding netdev is in the DOWN state after a vxlan vport has been successfully created.

+1 (if that is easy)

**Comment by awh** [comment]
Closed by f6864088082368f76ab940bb7e94541e613eee72
